stubs/setupVars.conf

```diff
@@ -0,0 +1,3 @@
+piholeInterface=eth0
+IPv4_address=192.168.1.100/24
+IPv6_address=fe80::bc3b:7855:7c13:4f84
```

Comment:
Note that these are changed in the current `development` version of pi-hole/pi-hole

piholeInterface -> PIHOLE_INTERFACE
IPv4_address -> IPV4_ADDRESS
IPv6_address -> IPV6_ADDRESS
piholeDNS1 -> PIHOLE_DNS_1
piholeDNS2 -> PIHOLE_DNS_2